docker_plugin/docker_client.py

```diff
@@ -33,7 +34,23 @@ def get_client(daemon_client):
     """
 
     try:
-        return Client(**daemon_client)
+        ctx.logger.info("Daemom connection options: {0}.".format(
+            str(daemon_client)))
+        tp = daemon_client.get('transport_protocol', 'tcp')
+        docker_host = daemon_client.get('host_string', 'localhost')
+        docker_port = daemon_client.get('host_port', 2375)
```

Comment:
Why do you need to specify the default value if it is already part of the plugin default values? Shouldn't it be:
```
docker_host = daemon_client.get('host_string')
docker_port = daemon_client.get('host_port')
```